pmm/pmm2-server-autobuild.groovy

```diff
@@ -97,23 +97,29 @@ pipeline {
             steps {
                 withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', accessKeyVariable: 'AWS_ACCESS_KEY_ID', credentialsId: 'pmm-staging-slave', secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']]) {
                     sh '''
-                        set -o errexit
+                        sg docker -c "
+                            set -o errexit
+
+                            aws ecr-public get-login-password --region us-east-1 | docker login -u AWS --password-stdin public.ecr.aws/e7j3v3n0
```

Comment:
Why did we add this?

The same question about `sg docker -c`

Reply:
I understood we wanted to migrate the login to AWS like we already do in https://github.com/Percona-Lab/jenkins-pipelines/blob/5032d130e6eff49049e700334391fbf9382007bc/pmm/pmm2-submodules.groovy#L233

Wasn't it the intention?

Reply:
Reverted